docker-for-mac/release-notes.md

```diff
@@ -13,12 +13,38 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.4.0
-2021-06-09
+## Docker Desktop 3.5.0
+2021-06-23
 
 [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
 [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
 
+
+### New
+
+- Dev Environments
+
+### Upgrades
+- [Compose V2 beta](https://github.com/docker/compose-cli/releases/tag/v2.0.0-beta.4)
```

Comment:
Should we not list the changes here, as a sub-list?
I remember Donnie asking about that.

Reply:
It's more than listing the changes. Donnie's asked if we can include how the changes introduced in an upgraded version impact Docker Desktop users. This isn't just for Compose, but for all other upgrades listed in this section. Perhaps we can leave it like this for this release and update it retrospectively

Reply:
He did. At least anything significant.

Reply:
There's a change mentioned below that references the compose-cli. I'll put it in a subsection.